pkgs/tools/security/tracee/default.nix

```diff
@@ -0,0 +1,88 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+
+, llvmPackages_13
+, pkg-config
+
+, zlib
+, libelf
+}:
+
+buildGoModule rec {
+  pname = "tracee";
+  version = "unstable-2022-03-10";
+
+  src = fetchFromGitHub {
+    owner = "aquasecurity";
+    repo = pname;
+    rev = "bc05bc0f9caf3e7d32ea8021c28fb6256ee3a799";
+    sha256 = "sha256-Prt/jAlRqMCGqiC83jnfLQrdGxv55r8g9JJCCrxvYCA=";
+    fetchSubmodules = true;
+  };
+  vendorSha256 = "sha256-I+d03IEbHqiZ2haXs96v1IUN1oSdZFJf2agTFREx9MM=";
+
+  strictDeps = true;
+  nativeBuildInputs = [ pkg-config llvmPackages_13.clang ];
+  buildInputs = [ zlib libelf ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    mkdir path
+    export PATH="$PATH:$PWD/path"
+
+    # the Makefile checks for git but we don't actually need it
+    touch ./path/git; chmod +x ./path/git
+
+    # also make clang-unwrapped easily available
+    ln -s ${llvmPackages_13.clang-unwrapped}/bin/clang ./path/clang-unwrapped
+
+    # VERSION="v${version}"
+    make CMD_CLANG_UNWRAPPED="clang-unwrapped"
+
+    runHook postBuild
+  '';
+
+  patches = [
+    # allow for building just bpf-core with clang-unwrapped since it includes
+    # -target bpf that is incompatible with clang-wrapped
+    # clang-unwrapped can't be used for the whole build since it wont find the
+    # necessary headers
+    ./bpf-core-clang-unwrapped.patch
+    # add -s to ldflags for smaller binaries
+    ./disable-go-symbol-table.patch
+  ];
+
+  doCheck = false;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,usr/share/tracee}
+
+    cp ./dist/tracee-ebpf $out/bin
+    cp ./dist/tracee-rules $out/bin
+
+    cp -r ./dist/rules $out/usr/share/tracee/
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://aquasecurity.github.io/tracee/latest/";
+    changelog = "https://github.com/aquasecurity/tracee/releases/tag/v${version}";
+    description = "Linux Runtime Security and Forensics using eBPF";
+    longDescription = ''
+      Tracee is a Runtime Security and forensics tool for Linux. It is using
+      Linux eBPF technology to trace your system and applications at runtime,
+      and analyze collected events to detect suspicious behavioral patterns. It
+      is delivered as a Docker image that monitors the OS and detects suspicious
+      behavior based on a pre-defined set of behavioral patterns.
+    '';
+    license = licenses.asl20;
+    maintainers = with maintainers; [ jk ];
+    platforms = platforms.linux;
+    # platforms = [ "x86_64-linux" ];
```

Comment:
```suggestion
```

Reply:
I'll be swapping it to just x86_64 linux as I had no luck with aarch64